taskcluster/ci/check/kind.yml

```diff
@@ -0,0 +1,35 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+kind-dependencies:
+    - docker-image
+
+transforms:
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+jobs:
+    ruff:
+        description: "ruff-py39"
+        run-on-tasks-for: ["action", "github-pull-request", "github-push"]
```

Comment:
`action` here seems wrong?

Reply:
Good catch
🤔 does that mean tox/unit tests should also not have "action" listed?

Reply:
It's probably a no-op. The `run-on-tasks-for` is only used if the target tasks method uses it, and actions don't even use target tasks.

So I'd say it's cleaner to remove both of them.